src/extension.ts

```diff
@@ -69,9 +71,51 @@ function npmCheck() {
   });
 }
 
+function getPackageManagerConfig(): PackageManagerConfiguration {
+  const config = vscode.workspace.getConfiguration().inspect('npm.packageManager');
+  const globalConfig = config?.globalValue;
+  const workspaceConfig = config?.workspaceValue;
+
+  const pickValueFromConfig = (config: unknown): PackageManagerConfiguration => {
+    if (typeof config !== 'string' || (config !== 'yarn' && config !== 'npm')) {
+      return undefined;
+    }
+
+    return config;
+  };
+
+  return pickValueFromConfig(workspaceConfig) || pickValueFromConfig(globalConfig);
+}
+
+function setPackageManagerConfig(value: PackageManagerConfiguration) {
+  vscode.workspace.getConfiguration().update('npm.packageManager', value, false);
```

Comment:
The 3rd parameter of `update()` function is documented like this :
- If `true` updates [Global settings](#ConfigurationTarget.Global).
- If `false` updates [Workspace settings](#ConfigurationTarget.Workspace).

Reply:
Yes, Got it.

Reply:
can be merged right?

Reply:
This feature is ready. Did you test it?

Reply:
Yes it's good